### PR TITLE
Fix status codes for error messages in README

### PR DESCRIPTION
**Pull request by schmijos**
I tried LHC against 1xx responses. It works :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ You can also access response data directly through the response object (with squ
 
 ## Exceptions
 
-Anything but a response code indicating success (2xx) raises an exception.
+Anything but a response code indicating success (1xx-3xx) raises an exception.
```
Comment (spape) [inline README.md]: 3xx is raising exception. see other comment.

File: README.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ Other methods are available using `LHC.request(options)`.
 
 ## Request
 
-The request class handles the http request, implements the interceptor pattern, loads configured endpoints, generates urls from url-templates and raises [exceptions](#exceptions) for any response code that is not indicating success (2xx).
+The request class handles the http request, implements the interceptor pattern, loads configured endpoints, generates urls from url-templates and raises [exceptions](#exceptions) for any response code that is not indicating success (4xx-5xx).
```
Comment (spape) [inline README.md]: 3xx raises exception, see other comment.

**Comment by spape** [comment]
How about raising 1xx ? What do expect an http client to do on 101 ?!

**Comment by schmijos** [comment]
> LHC.get('http://local.ch')

LHC.get('http://local.ch', followlocation: true)

**Comment by spape** [comment]
The documentation describes the behavior without `followlocation: true` obviously... I know the standard around 101, I know how browsers (one kind of http client) handles those, but again, how would you expect a ruby http client to work? We can't do streaming, so for me, only an exception message would be an option, making your documentation changes redundant.

**Pull request closed without merging**